src/objective-c/examples/SwiftSample/ViewController.swift

```diff
@@ -70,18 +70,18 @@ class ViewController: UIViewController {
 
     let requestsWriter = GRXWriter(value: request.data())
 
-    let call = GRPCCall(host: RemoteHost, path: method.HTTPPath, requestsWriter: requestsWriter)
+    let call = GRPCCall(host: RemoteHost, path: method?.httpPath, requestsWriter: requestsWriter)
 
-    call.requestHeaders.setObject("My value", forKey: "My-Header")
+    call?.requestHeaders.setObject("My value", forKey: "My-Header" as NSCopying)
```

Comment:
Same with `call?.`; `GRPCCall` should never return `nil` under normal circumstances, so force it with `!` on initialization.